docs/03-extensions/stripe/how-to/customize-data.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Customize data sent to Stripe
+description: "This guide explains how to Customize data sent to Stripe"
+sidebar_position: 1
+---
+
+<p>{frontMatter.description}</p>
+
+:::caution WIP
+
+<span>
+  This advanced pattern must be documented with further details. While we are
+  working on it, please <ContactLink /> if you need further assistance.
+</span>
+
+:::
+
+The Stripe payment module is extensible. It leverages Front-Commerce's "data
+transform" pattern to allow developers to customize payloads sent to Stripe for
+Customer and Cart content.
```

Comment:
which "data transform" pattern? do you have a link to it?

Reply:
added link to relevant doc